ontology/yaml/resources/HVAC/entity_types/RP.yaml

```diff
@@ -37,4 +37,30 @@ RP_DPBHCC_FTC:
   implements:
   - RP
   - DPBHCC
-  - FTC
\ No newline at end of file
+  - FTC
+
+#Entity Types for DE-FRA-NEM32
+
+RP_CAM_DPHC2_ZTC:
+  description: "Radiant panel for room heating and cooling, Two pipe installation."
+  is_canonical: true
+  implements:
+  - RP
+  - CAM
+  - DPHC2
+  - ZTC
+  uses:
+  - cooling_output_percentage_command
```

Comment:
wouldnt these be optional under whatever cooling/heating types youve implemented? valve positions essentially